content/en/docs/reference/setup-tools/kubeadm/cri-installation.md

```diff
@@ -0,0 +1,144 @@
+---
+reviewers:
+- vincepri
+title: CRI Installation
```

Comment:
please lowercase `Installation` i think we should have this consistent for other pages too.